skypy/utils/tests/test_completeness_function.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+
+def test_logistic_completeness_function():
+    from skypy.utils.completeness_function import logistic_completeness_function
+
+    # Test that (nb, nm) array is returned if m in (mb, nm) and m95 and m50 in (nb, ) are given
+    m = np.ndarray((5, 50))
+    m[:] = np.linspace(18, 28, 50)
+    m = m.T
```

Comment:
`np.reshape` is much better for this
https://numpy.org/doc/stable/reference/generated/numpy.reshape.html